src/sage/misc/superseded.py

```diff
@@ -5,16 +5,15 @@
 add a deprecation warning. This will be shown once, the first time
 that the deprecated function is called.
 
-Note that all doctests in the following use the trac ticket number
-:trac:`13109`, which is where this mandatory argument to
+Note that all doctests in the following use the github issue number
+:github:`13109`, which is where this mandatory argument to
```

Comment:
Does the `:github:` role already exist?

Reply:
I added it as part of this PR (by changing `src/tox.ini` and `src/sage/misc/sagedoc.py`), but based on the discussion below and consulting with some people at SD117, I'm changing it to `:issue:`.